### PR TITLE
Fixing navigation

### PR DESCRIPTION
**Pull request by Andertech**
This addresses small navigation issues, adds docs resources.

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ params:
   AuthorName: "VividCortex"
   GitHubUser: "VividCortex"
   GTMID: "6NVF"
-  documentation: "/docs/"
+  documentation: "https://docs.vividcortex.com/"
```
Comment (mtrias) [inline config.yaml]: is this needed? sounds like it is duplicated with baseurl ?

File: themes/cortex-docs/layouts/index.html [hunk 2]
```diff
@@ -1,9 +1,60 @@
 {{ partial "head.html" . }}
 
-<section role="main">
+<div class="docs-heading">
+    <div class="wrp">
+        <a href="/docs/" class="docs-heading__tit" title="go to documentation home">Documentation</a>
```
Comment (mtrias) [inline themes/cortex-docs/layouts/index.html]: The URL here should be "/"

File: themes/cortex-docs/layouts/section/getting-started.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{{ partial "head.html" . }}
+
+<div class="docs-heading">
+    <div class="wrp">
```
Comment (mtrias) [inline themes/cortex-docs/layouts/section/getting-started.html]: seems that this file is very similar to `how-to-use-vividcortex.html` and also to `general-reference` can they be just merged into one?

Comment (Andertech) [inline themes/cortex-docs/layouts/section/getting-started.html]: I was thinking of having a different page for each section with different content in them, I haven't thought of anything witty just yet.

Comment (mtrias) [inline themes/cortex-docs/layouts/section/getting-started.html]: what would that different content look like?

Comment (Andertech) [inline themes/cortex-docs/layouts/section/getting-started.html]: A list depicting pages for each section and maybe a small description of each. Nothing out of the ordinary.

Comment (mtrias) [inline themes/cortex-docs/layouts/section/getting-started.html]: shouldn't that be just a separate partial to include, instead of a different layout ?

**Pull request merged**